### DISCUSSION TITLE
pages/UA-PNP/Docs-as-Code/topic1/

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#12] [Q&A]
# pages/UA-PNP/Docs-as-Code/topic1/

A next-gen documentation site for HCL <product name>.

https://github01.hclpnp.com/pages/UA-PNP/Docs-as-Code/topic1/

**Comment by vishwanathan-m1** [comment]
Image needs to be formatted.

**Comment by vishwanathan-m1** [comment]
Image Test

**Comment by vishwanathan-m1** [comment]
<img width="125" alt="blur_image" src="https://github.com/HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home/assets/83331933/29208a47-bffc-4cb3-9bc3-eee6c385b39e">

Please check the above images.

**Reply by vishwanathan-m1** [reply]
Font Test1

**Comment by vishwanathan-m1** [comment]
Test Comment

**Reply by vishwanathan-m1** [reply]
ok
